prelude/runtime/palloc.c

```diff
@@ -690,14 +696,6 @@ void SKIP_print_persistent_size() {
 }
 
 void* sk_palloc(size_t size) {
-  if (ginfo->fileName == NULL) {
```

Comment:
Will this still work with `__APPLE__` defined?

Reply:
Good question. OSX support is currently broken for other reasons (and has not been an explicit goal so far). I will try to revive it once current priorities are settled.